testdata/artifacts/package/example/0.0.2/docs/README.md

```diff
@@ -0,0 +1,22 @@
+[role="xpack"]
```

Comment:
Couldn't you reuse here one of the existing test package? https://github.com/elastic/package-registry/tree/master/testdata/package

Reply:
I think this is possible now as I changed the way I compare package content (compare listings instead of created archive which was different depending on the platform).

Reply:
Lets use an existing one if possible. If we can't lets at least use the same path to have all test packages in one place and then I would remove quite a few of the assets to just have 1-2 files per type.

Reply:
Fixed